antora/spec/modules/ROOT/pages/index.adoc

```diff
@@ -6,47 +6,150 @@
 include::{config}/attribs.adoc[]
 include::{generated}/specattribs.adoc[]
 
-
 == About
 
-link:https://docs.vulkan.org/[This site] gathers together several key Vulkan
-documents including specifications, extension proposals, guides, tutorials
-and samples into a single site.
-This allows searching and cross-linking across documents, to help navigate
-quickly to the information you need for developing Vulkan-based
-applications.
-
-For more information about and resources for using the Vulkan graphics API,
-see the Vulkan developer website at https://vulkan.org .
+Welcome the the official Documentation site for the cross-platform Vulkan
+graphics and compute API.
+This is your starting point for all things related to
+https://www.vulkan.org[Vulkan].
+This site gathers together several key Vulkan documents into a convenient
+single site.This allows searching and cross-linking across documents, to
+help navigate quickly to the information you need for developing
+Vulkan-based applications.
 
 This build of the site includes the Vulkan {SPECREVISION} API specification
 {APITITLE}, generated on {SPECDATE} {SPECREMARK}.
 
+== How to navigate this site
```

Comment:
```suggestion
== How to Navigate This Site
```

Style for titles is to capitalize everything except short words, see https://registry.khronos.org/vulkan/specs/latest/styleguide.html#writing-titlecase (applies to other changed titles as well).